lib/location_settings.dart

```diff
@@ -17,6 +17,7 @@ class LocationSettings {
   final LocationAccuracy accuracy;
   final int interval; //seconds
   final double distanceFilter;
+  final String channelName;
```

Comment:
I think `notificationChannelName` is better name for this var.

Reply:
Ok, I've changed it.